src/DvbData.h

```diff
@@ -284,7 +285,9 @@ class Dvb
   KVStore m_kvstore;
   Settings m_settings;
 
-  P8PLATFORM::CMutex m_mutex;
+  std::atomic<bool> m_running = {false};
+  std::thread m_thread;
+  std::mutex m_mutex;
```

Comment:
You can try a `std::recursive_mutex` know from some cases that a small `std::mutex` can be problematic if call from different places and produced a freeze (think was on vfs.rar where I'm seen freeze problems and why I used the recursive).

Reply:
It was ok with the std::mutex, just had to move lock to the calling function.